resources/todomvc/architecture-examples/jquery/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "todomvc-jquery",
   "private": true,
   "scripts": {
+    "dev": "http-server . -p 7001 -c-1 --cors",
```

Comment:
not a fix here, but to be consistent with other framework scripts, added it here...

Reply:
yeah that's useful :-) thanks